lib/binarisYML.js

```diff
@@ -30,10 +30,16 @@ const saveBinarisConf = async function saveBinarisConf(funcDirPath, binarisConf)
 
 const getFunctionsSection = function getFunctionsSection(binarisConf) {
   // ensure configuration has the expected field
-  if (!Object.prototype.hasOwnProperty.call(binarisConf, funcStr)) {
+  if (typeof binarisConf !== 'object' ||
+      binarisConf === null ||
+      !Object.prototype.hasOwnProperty.call(binarisConf, funcStr)) {
     throw new Error(`Your ${binarisConfFile} did not contain a require field: <${funcStr}>`);
```

Comment:
nit: "require*d*".

For that matter, why are we logging errors in the past tense, with pronouns?  I think I'm more used to seeing e.g. "`${binarisConfFile} is missing required field <${funcStr}>`" or similar.  (But these can maybe wait for a separate PR.)